packages/expo-navigation-bar/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- Restricted color types to string to prevent the use of illegal color types (PlatformColor) untill supported. ([#34053](https://github.com/expo/expo/pull/34053) by [@chrfalch](https://github.com/chrfalch))
```

Comment:
```suggestion
- Restricted color types to string to prevent the use of illegal color types (PlatformColor) until supported. ([#34053](https://github.com/expo/expo/pull/34053) by [@chrfalch](https://github.com/chrfalch))
```